### PR TITLE
Create library to fetch OONI measurements

### PR DESCRIPTION
**Pull request by fortuna**
This creates a library and tool to efficiently fetch measurements for a given country and time range.
It fetches data directly from the S3 bucket, bypassing the performance issues of the API.
Fetching is parallelized in order to not have I/O and CPU blocking on each other.
This only supports the new `ooni-data-eu-fra` bucket. Support for the old bucket is a TODO.
This PR removes the old API code, since it's not really that useful.

/cc @bassosimone @hellais

### CODE REVIEW

File: netanalysis/ooni/README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    python -m netanalysis.ooni.data.fetch_measurements --country=BY --output_dir=./ooni_data/
+
+If you call it a second time, it will redownload data already downloaded.
+
+Use `--first_date` and `--last_date` to restrict the fetch to a specific, inclusive, date range. For example:
+
+    python -m netanalysis.ooni.data.fetch_measurements --output_dir=./ooni_data/ --country=BY --first_date=2021-01-01 --last_date=2021-01-31
+
+### Data trimming
+By default the tool will drop any measurement field that are longer than 1000 characters in order to save space. You can change that dy setting a different value for `--max_string_size`.
```
Comment (ohnorobo) [inline netanalysis/ooni/README.md:18]: delete 'dy'

File: netanalysis/ooni/README.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+This is primarily intended to drop the response bodies, which are often not needed and takes most of the space. For the date range example above, we download 158 MiB of data, but only store 18 MiB after the trimming, a nearly 9x difference!
```
Comment (ohnorobo) [inline netanalysis/ooni/README.md:20]: takes -> take

**Review by ohnorobo — APPROVED** [review]

File: netanalysis/ooni/README.md (hunk 2 above)
Comment (fortuna) [reply netanalysis/ooni/README.md]: Fixed

File: netanalysis/ooni/README.md (hunk 1 above)
Comment (fortuna) [reply netanalysis/ooni/README.md]: Fixed

**Pull request merged**